ONSdigital/eq-survey-runner
# eq 497 grid and eq 498 header

### What is the context of this PR?

Moved eq-scss grid into project and updated header in basetheme
### How to review
1. Checkout this branch
2. npm start
3. Use the schema 0_basetheme.json
4. Complete the survey and check the grid and header match designs 
### Who worked on the PR

@bennpearson
